src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -488,6 +488,34 @@ namespace Akka.Actor
         public override int CompareTo(Akka.Actor.ActorPath other) { }
         public override Akka.Actor.ActorPath WithUid(long uid) { }
     }
+    public sealed class CoordinatedShutdown : Akka.Actor.IExtension
+    {
+        public const string PhaseActorSystemTerminate = "actor-system-terminate";
```

Comment:
The constants should be `public`, as they're useful when an end-user wants to append a task to a built-in phase. The constants in this case are all of the phase names.